services/exam-manager/app/main.py

```diff
@@ -54,11 +54,11 @@ async def startup():
             status_code=500,
             detail="SQL-DB: Device table is required but does not exist.",
         )
-    if "workflow" not in tables:
-        raise HTTPException(
-            status_code=500,
-            detail="SQL-DB: Workflow table is required but does not exist.",
-        )
+    # if "workflow" not in tables:
+    #     raise HTTPException(
+    #         status_code=500,
+    #         detail="SQL-DB: Workflow table is required but does not exist.",
+    #     )
```

Comment:
if check guard is not enabled on purpose?

Reply:
This is inside the startup routine of the exam-manager and was used to check if the workflow table (created by the workflow-service) has been created successfully. As this table is obsolete now and has been removed (we only have the workflow table created by the exam-manager, which was the Job table previously) this check is not feasible anymore. Instead we check the existence of the workflow-table in the health check endpoint.